src/code_8015080.c

```diff
@@ -4,8 +4,8 @@
 #include "input.h"
 #include "memory.h"
 #include "menu_input.h"
-#include "text1.h"
-#include "text2.h"
+#include "text.h"
```

Comment:
A few double `text.h` includes that need to be cleaned up.

Reply:
I fixed these two. Are there any in other files left?